var/spack/repos/builtin/packages/xios/package.py

```diff
@@ -0,0 +1,224 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+from spack import *
+import os
+
+
+class Xios(Package):
+    """XML-IO-SERVER library for IO management of climate models."""
+
+    homepage = "https://forge.ipsl.jussieu.fr/ioserver/wiki"
+    url = "https://forge.ipsl.jussieu.fr/ioserver"
+
+    version('1.0', revision=910, 
+            svn='http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-1.0')
+    version('develop', svn='http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/trunk')
+
+    variant('mode', values=('debug', 'dev', 'prod'), default='dev',
+            description='Build for debugging, development or production')
+    # NOTE: oasis coupler could be supported with a variant
+
+    # Use spack versions of blitz and netcdf for compatibility
+    # with recent compilers and optimised platform libraries:
+    patch('bld_extern_1.0.patch', when='@:1.0')
+    patch('bld_extern_1.x.patch', when='@1.1:')
+
+    # Workaround bug #17782 in llvm, where reading a double
+    # followed by a character is broken (e.g. duration '1d'):
+    patch('llvm_bug_17782.patch', when='@1.1: %clang')
+
+    depends_on('netcdf+mpi')
+    depends_on('netcdf-fortran^netcdf+mpi')
```

Comment:
you don't need `^netcdf+mpi`, it will be part of the graph of dependencies automatically as you have it above anyway.

Reply:
Thanks for pointing that out. I've just pushed an update.